### PR TITLE
Fix mortar ammo handling init on dedicated

### PR DESCRIPTION
**Pull request by SzwedzikPL**
**When merged this pull request will:**
- Change mortarInit into global event with turret locality check to be sure that mortar init is executed on machine that owns turret.
- Fix for https://github.com/acemod/ACE3/issues/3557
- Tested on dedicated multiplayer

### CODE REVIEW

**Comment by esteldunedain** [comment]
:+1: for merging

**Comment by commy2** [comment]
>Tested on dedicated multiplayer
Nice. GJ man!

File: addons/mk6mortar/functions/fnc_mortarInit.sqf [hunk 1]
```diff
@@ -18,27 +18,28 @@
...
     } forEach magazines _mortar;
 };
 
-// Replace current weapon with ammo handling weapon
+// Replace current turret weapon with ammo handling weapon
 private _currentWeapon = _mortar weaponsTurret [0] select 0;
 private _newWeapon = "";
 
-if (_currentWeapon == "mortar_82mm") then {
+if (tolower _currentWeapon == "mortar_82mm") then {
```
Comment (commy2) [inline addons/mk6mortar/functions/fnc_mortarInit.sqf]: technically not needed, because `==` compares strings case insensitive, but whatever

Comment (jonpas) [inline addons/mk6mortar/functions/fnc_mortarInit.sqf]: It should also be `toLower` if we're being picky.

Comment (commy2) [inline addons/mk6mortar/functions/fnc_mortarInit.sqf]: ironic, isn't it?

Comment (jonpas) [inline addons/mk6mortar/functions/fnc_mortarInit.sqf]: :ice_cream:

**Comment by commy2** [comment]
branch needs update. can't do that. @SzwedzikPL pls fix. also remove that tolower while you're at it. thanks.

**Pull request merged**